### PR TITLE
test: support Redis integration testing

### PR DESCRIPTION
**Pull request by funky-eyes**
- [ ] I have registered the PR [changes](../changes).

### CODE REVIEW

File: server/src/test/java/org/apache/seata/server/session/redis/RedisDistributedLockerTest.java [hunk 1]
```diff
@@ -122,7 +121,7 @@ public void test_acquireLock_concurrent() {
 
        //sleep 60s
         try {
-            Thread.sleep(2000);
+            Thread.sleep(2100);
```
Comment (liuqiufeng) [inline server/src/test/java/org/apache/seata/server/session/redis/RedisDistributedLockerTest.java]: Why modify the sleep time?

File: server/src/test/java/org/apache/seata/server/session/redis/RedisDistributedLockerTest.java [hunk 2]
```diff
@@ -40,6 +41,7 @@
  *
  */
 @SpringBootTest
+@EnabledIfSystemProperty(named = "redisCaseEnabled", matches = "false")
```
Comment (liuqiufeng) [inline server/src/test/java/org/apache/seata/server/session/redis/RedisDistributedLockerTest.java]: Whether this should be true

File: config/seata-config-core/src/test/java/org/apache/seata/config/FileConfigurationTest.java [hunk 3]
```diff
@@ -48,13 +51,14 @@ void addConfigListener() throws InterruptedException {
...
-            System.out.println("oldValue:" + event.getOldValue() + ",newValue:" + event.getNewValue());
+            logger.info("dataId: {}, oldValue: {}, newValue: {}", event.getDataId(), event.getOldValue(),
+                event.getNewValue());
             countDownLatch.countDown();
         });
         System.setProperty(dataId, String.valueOf(!value));
-        countDownLatch.await(10, TimeUnit.SECONDS);
-        System.out.println(fileConfig.getBoolean(dataId));
-        System.out.println(value);
+        countDownLatch.await();
```
Comment (funky-eyes) [inline config/seata-config-core/src/test/java/org/apache/seata/config/FileConfigurationTest.java]:
```suggestion
        countDownLatch.await(60, TimeUnit.SECONDS);
```

**Review by liuqiufeng — APPROVED** [review]
LGTM

**Pull request merged**